styles/styles.css

```diff
@@ -1,51 +1,241 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
 *{
     box-sizing: border-box;
     margin: 0;
 }
 
-.body {
-    background-color: #ffffff;
-    height: 570px;
+body {
+}
+
+:root {
+    --columns-count: 14;
+    --columns-count-m: 6;
+}
+
+.header {
+    /*background-image: url("../images/castelmezzano.png");*/
+    background: url("../images/castelmezzano.png") center no-repeat;
+    background-size: cover;
+    position: relative;
+    height: 100vh;
+    width: 100%;
+    max-height: 900px;
+    /*max-width: 1440px;*/
+    margin: 0 auto;
+}
+
+.header-background {
+    background-color: rgba(18, 18, 18, 0.25);
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
 }
 
-hr {
-    color: #F5BD41;
+.section-header {
+    padding: 51px 104px 120px 104px;
+}
+
+.triphouse{
+    /*padding: 51px 104px 0 104px;*/
+    max-width: 1232px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 auto;
+}
+
+.triphouse-right {
```

Comment:
try to not use classes with names like this. You can choose names like link__header-wrapper, or something similar. It's not an error, just a proposal.